### DISCUSSION TITLE
Can I add a Save As button in the shortcut toolbar?

### DISCUSSION THREAD

**Discussion by ffs-akl** [tsujan/FeatherPad#828] [Q&A]
I've hidden the menu, but every time I need to save a file as another file name or other location . I'll have to go to the menu and look for it.  Could we add it to the shortcut toolbar?
![swappy-20241011-105409](https://github.com/user-attachments/assets/1d578650-491a-449c-83b9-b91b65d00beb)

**Comment by tsujan** [comment]
> I've hidden the menu

When you hide the menu-bar, a menu button appears on the right side of the tool-bar. It shows all the menus when clicked.

> every time I need to save a file as another file name or other location

That's called "Save As" (in English), and its shortcut is `Ctrl+Shift+S` by default. The shortcut can be changed in the Preferences dialog.

**Reply by ffs-akl** [reply]
Maybe I didn't express myself clearly, what I meant is as the following photo
![swappy-20241011-213742](https://github.com/user-attachments/assets/ea02c1c7-e9b4-4b7d-89a3-e8685f87fc14)
If you don't think this is helpful to the user experience, please feel free to ignore it.

**Reply by tsujan** [reply]
> what I meant is as the following photo

I've been cautious about adding buttons to the toolbar, because I want to keep it as clean and functional as possible. If a Save-As button was added next to the Save button, they could be easily mistaken for each other and damage UX. Adding a separator between them would clutter the toolbar. Such a situation doesn't exist with menus or shortcuts.

`Ctrl+Shift+S` is a universal shortcut for Save-As in editors, and all users are familiar with it.
